Mathlib/MeasureTheory/Decomposition/SignedLebesgue.lean

```diff
@@ -178,20 +178,16 @@ def rnDeriv (s : SignedMeasure α) (μ : Measure α) : α → ℝ := fun x =>
     (s.toJordanDecomposition.negPart.rnDeriv μ x).toReal
 #align measure_theory.signed_measure.rn_deriv MeasureTheory.SignedMeasure.rnDeriv
 
--- Porting note: The generated equation theorem is the form of `rnDeriv s μ x`.
```

Comment:
Isn't this still true?

Reply:
Yes, but I don't see any need to preserve this information.

Reply:
this explains why we need the `_def` version at all, and also flags a perceived usability shortcoming in the default equation lemmas.

Reply:
Okay, I restored it as a plain comment.